src/Files.App/ViewModels/UserControls/Widgets/RecentFilesWidgetViewModel.cs

```diff
@@ -68,6 +68,7 @@ public RecentFilesWidgetViewModel()
 		public async Task RefreshWidgetAsync()
 		{
 			IsRecentFilesDisabledInWindows = !CheckIsRecentItemsEnabled();
+			Items.Clear();
```

Comment:
The recent files widget already has a watcher that updates the list, is there any reason to manually clear the list?

Reply:
There's just one case where that is needed: when you clear `File Explorer` history (`File Explorer>Options>Privacy>Clear`).
In this scenario the watcher doesn't work and items remain visible after updating the home page

Reply:
It works on my end, but there is a known issue where it doesn't work for some users. We're tracking this in #16658.